### DISCUSSION TITLE
Private Notes or Annotations in Charts

### DISCUSSION THREAD

**Discussion by techroad-dev** [plausible/analytics#3108] [Anything Else]
The product is great, but it lacks one essential feature: the ability to add private notes.

As the owner of multiple websites, I'm constantly optimizing my pages to follow the best SEO practices. However, there's no way to add a private note or annotation to a specific date so that I can track the performance of my pages over time.

This is a critical feature that would be incredibly helpful. For example, if I make a change to a page and it results in a drop in traffic, I can easily revert the change by referring to my private notes. I hope the developers will consider adding this feature in the future. It would be a valuable addition to an already great product.

**Comment by metmarkosaric** [comment]
thanks @techroad-dev! there's a thread for this already here: https://github.com/plausible/analytics/discussions/97
